test-app/tests/integration/components/checkbox-group-field-test.gts

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+
+import { click, render } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import CheckboxGroupField from '@crowdstrike/ember-toucan-core/components/form/checkbox-group-field';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+module('Integration | Component | CheckboxGroupField', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      <CheckboxGroupField @label="Label" @name="group" data-group-field />
+    </template>);
+
+    assert.dom('[data-group-field]').exists();
+    assert.dom('[data-group-field]').hasNoAttribute('aria-invalid');
+
+    assert.dom('[data-label]').hasText('Label');
+  });
+
+  test('it renders yielded CheckboxFields', async function (assert) {
+    await render(<template>
+      <CheckboxGroupField
+        @label="Label"
+        @name="group"
+        data-group-field
+        as |group|
+      >
+        <group.CheckboxField
+          @label="option-1"
+          @option="option-1"
+          data-checkbox-1
+        />
+        <group.CheckboxField
+          @label="option-2"
+          @option="option-2"
+          data-checkbox-2
+        />
+      </CheckboxGroupField>
+    </template>);
+
+    assert.dom('[data-group-field]').exists();
+    assert.dom('[data-checkbox-1]').exists();
+    assert.dom('[data-checkbox-2]').exists();
+  });
+
+  test('it sets the `value` attribute of the yielded CheckboxFields based on the provided `@option` argument', async function (assert) {
+    await render(<template>
+      <CheckboxGroupField
+        @label="Label"
+        @name="group"
+        data-group-field
+        as |group|
+      >
+        <group.CheckboxField
+          @label="option-1"
+          @option="option-1"
+          data-checkbox-1
+        />
+        <group.CheckboxField
+          @label="option-2"
+          @option="option-2"
+          data-checkbox-2
+        />
+      </CheckboxGroupField>
+    </template>);
+
+    assert.dom('[data-checkbox-1]').hasValue('option-1');
+    assert.dom('[data-checkbox-2]').hasValue('option-2');
+  });
+
+  test('it curries the `name` attribute to yielded CheckboxFields from the `@name` argument', async function (assert) {
+    await render(<template>
+      <CheckboxGroupField
+        @label="Label"
+        @name="group"
+        data-group-field
+        as |group|
+      >
+        <group.CheckboxField
+          @label="option-1"
+          @option="option-1"
+          data-checkbox-1
+        />
+        <group.CheckboxField
+          @label="option-2"
+          @option="option-2"
+          data-checkbox-2
+        />
+      </CheckboxGroupField>
+    </template>);
+
+    assert.dom('[data-checkbox-1]').hasAttribute('name', 'group');
```

Comment:
Suggestion: similar thing here...can we change the `@name` to something other than `group` because it can be confused with `|group|` when scanning the code.

Reply:
Went ahead and updated with https://github.com/CrowdStrike/ember-toucan-core/pull/81/commits/abb6338f6bfbc957e4335850e0d2bca591caab96 - thanks!